GiniVision/Classes/Core/Screens/Document picker/DocumentPickerCoordinator.swift

```diff
@@ -195,13 +195,6 @@ public final class DocumentPickerCoordinator: NSObject {
 
         }
         
-        // This is needed since the UIDocumentPickerViewController on iPad is presented over the current view controller
-        // without covering the previous screen. This causes that the `viewWillAppear` method is not being called
-        // in the current view controller.
-        if !device.isIpad {
-            setStatusBarStyle(to: .default)
```

Comment:
Why is this obsolete now?

Reply:
Thank for noticing that I've test it with the newest iOS and didn't have any changes in status bar behaviour , installed old simulations and found out that we still need for iOS 11, iOS 12